### ISSUE TITLE
Support for openGL 4

### ISSUE THREAD

**Issue by wbyates777** [ocornut/imgui#7854]
### Version/Branch of Dear ImGui:

No modifications

### Back-ends:

imgui_impl_opengl3.cpp and imgui_impl_sdl2.cpp

### Compiler, OS:

clang 15, Xcode, macOS  Sonoma 14.6

### Details:

Are there plans to provide backends for opengl 4

I'm using opengl 4.1 with the backend 'imgui_impl_opengl3.cpp' and although everything seems to work OK it would be nice to have an official version.

### Minimal, Complete and Verifiable Example code:

```cpp
// Here's some code anyone can copy and paste to reproduce your issue
ImGui::Begin("Example Bug");
MoreCodeToExplainMyIssue();
ImGui::End();
```

**Comment by PathogenDavid** [comment]
`imgui_impl_opengl3` is fully compatible with OpenGL 4, there's no need for a separate backend.

**Comment by ocornut** [comment]
What David says. There’s no value in using features that we don’t need.

**Comment by wbyates777** [comment]
No problem.On 2 Aug 2024, at 17:31, omar ***@***.***> wrote:﻿
Closed #7854 as completed.
